docs/usage/installation.md

```diff
@@ -2,32 +2,32 @@
 
 ## Local installation
 
-For [Node modules projects](https://docs.npmjs.com/getting-started/creating-node-modules) we recommend to install **semantic-release** locally and to run the `semantic-release` command with a [npm script](https://docs.npmjs.com/misc/scripts):
+For [Node modules projects](https://docs.npmjs.com/getting-started/creating-node-modules) we recommend to install **semantic-release** locally and to run the `semantic-release` command with [npx](https://www.npmjs.com/package/npx):
 
 ```bash
 $ npm install --save-dev semantic-release
 ```
 
-In your `package.json`:
-
-```json
-"scripts": {
-  "semantic-release": "semantic-release"
-}
-```
-
 Then in the CI environment:
 
 ```bash
-$ npm run semantic-release
+$ npx semantic-release
 ```
 
 ## Global installation
 
-For other type of projects we recommend to install **semantic-release** globally directly in the CI environment:
+For other type of projects we recommend to install **semantic-release** directly in the CI environment:
 
 ```bash
+# Option 1: npx
+$ npx semantic-release
+```
+
+```bash
+# Option 2: npm global install
```

Comment:
IT seems there is a problem with option 2 as we would install semantic-release globally and run it twice (once with `semantic-release` and once with `npx semantic-release`).

Overall we probably don't need two options but just `npx semantic-release` as `npx` would take care of the install.